plenum/test/storage/test_rocksdb_hash_store.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from ledger.compact_merkle_tree import CompactMerkleTree
+from ledger.ledger import Ledger
+from ledger.test.test_file_hash_store import nodesLeaves, \
+    generateHashes
+
+from plenum.persistence.rocksdb_hash_store import RocksDbHashStore
+
+
+@pytest.yield_fixture(scope="module")
+def rocksdbHashStore(tdir):
```

Comment:
Are there the same tests as for `LevelDbHashStore`? maybe just use the same test with parametrized fixtures?

Reply:
Done.